### DISCUSSION TITLE
The CPU has been soaring, even in the middle of the night

### DISCUSSION THREAD

**Discussion by horizon188** [nuxt/nuxt#22265] [Questions]
### Environment

"nuxt": "3.6.3"
node：v16.17.1

### Reproduction

<img width="914" alt="38f9cb235522f0e85ee0ef489dab0c01_" src="https://github.com/nuxt/nuxt/assets/37240607/eff19437-34d9-4e8a-8de7-2c913f250d81">

### Describe the bug

Since the early morning, it has been soaring, and there must have been no user access at that time. I don't know where it came from and what caused the CPU to continue to rise

### Additional context

How should I solve this problem?

**Comment by blacksheep107** [comment]
Maybe a memory leak

**Reply by horizon188** [reply]
At present, there is no such situation in the testing environment, only in the production environment. I don't know how to locate the problem, what should I do?

**Reply by saifobeidat** [reply]
We once faced a memory leak on production due to a wrong usage of the plugins (putting variables outside the plugin function)

Check these:
https://stackoverflow.com/questions/66831561/nuxt-add-global-plugins-memory-leakage-issue

https://medium.com/le-collectionist/solving-server-side-memory-leaks-on-nuxt-js-ecd6b16b54a0

Debugging memory leaks was challenging on production, maybe you need to reproduce it on dev env with fake traffic
